pkgs/tools/misc/geekbench/default.nix

```diff
@@ -15,12 +15,19 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ makeWrapper ];
 
   installPhase = ''
-    mkdir -p $out/bin
+    mkdir -p $out/bin $out/lib
     cp -r geekbench.plar geekbench5 geekbench_x86_64 $out/bin
 
+    # needed for compute benchmark
+    ln -s ${linuxPackages.nvidia_x11}/lib/libcuda.so $out/lib/
+    ln -s ${ocl-icd}/lib/libOpenCL.so $out/lib/
+    ln -s ${ocl-icd}/lib/libOpenCL.so.1 $out/lib/
+    ln -s ${vulkan-loader}/lib/libvulkan.so $out/lib/
+    ln -s ${vulkan-loader}/lib/libvulkan.so.1 $out/lib/
```

Comment:
Only noticing this after merging: Do you specify version 1 on purpose, or should this maybe link all versions instead?

Reply:
That's what geekbench5 tried to open: https://github.com/NixOS/nixpkgs/issues/87990#issuecomment-635540959

the opencl issue don't look related to a library missing https://github.com/NixOS/nixpkgs/issues/94982

do you think it could be an improvement?

Reply:
There is actually just one version, with different names.

```
ls -la ${ocl-icd}/lib/
-r-xr-xr-x 1 nobody nogroup    994 Jan  1  1970 libOpenCL.la
lrwxrwxrwx 1 nobody nogroup     18 Jan  1  1970 libOpenCL.so -> libOpenCL.so.1.0.0
lrwxrwxrwx 1 nobody nogroup     18 Jan  1  1970 libOpenCL.so.1 -> libOpenCL.so.1.0.0
-r-xr-xr-x 1 nobody nogroup 150576 Jan  1  1970 libOpenCL.so.1.0.0
dr-xr-xr-x 2 nobody nogroup   4096 Jan  1  1970 pkgconfig
```

```
ls -la ${vulkan-loader}/lib/
lrwxrwxrwx 1 nobody nogroup     14 Jan  1  1970 libvulkan.so -> libvulkan.so.1
lrwxrwxrwx 1 nobody nogroup     20 Jan  1  1970 libvulkan.so.1 -> libvulkan.so.1.2.131
-r-xr-xr-x 1 nobody nogroup 458168 Jan  1  1970 libvulkan.so.1.2.131
```

Reply:
Yes, the usual naming convention is that `libsomething.so` will link to the latest version, `libsomething.so.n` will link to version `n`. So in this case, it might break if `libvulkan` 2 is released.

Reply:
ok, i understand. but it will not improve the current situation if all files are copied.
i will add that later